vueui/dist/app.css

```diff
@@ -1,4 +1,4 @@
/uI19qx5jJqwP6oUhaBh+kWjNP4vccLGDOowvZovbHuAqwhEvxOIAYVj07lgbXKwiit9d/EyOClv8SqlGy8RAMEu9SzsD+fWNYX3RN4AMBz0RhRg0cErGZ0e73AExbx7wcQtMK52XrghBkt5jobMoY1vG5cFjvG/iNYsPnnoljaAi2voPeX15WoXlgVV7ljJb1MBUGCsfaMEOcBKVRTB1CRKXo94Nl/h9IT0m3jqM1jRfYlQX2wYAV+ZdWe74lmhD/Yll4AK+JcdHg/vMtL+NWQDG5aq5YZKnL6dQrPz9IADe1gEFu+YdPLaxMjWmEmgEt0rRmXquhz8TBBfjuoRRl8cTHEvkN18w9DrABS3zcC027f1KCmdL7esoiw8vHzKFgDdxnY+6M8n42yyKAV8BGFb6SqieH7Qs9K0XMk7ZWHXcrPL5lT5O+oyFZN8szFhi+YmfBeZSLBpKmivDuYIY2QEVzEyKrg6+kWgy118wVRPHeowAq/rhYwVtqZs3YjYw8L3AIYH9mWCaaOXli+En3lL3oHmJfoYCWRZ8HHzM7yv8uCobTUMWZaz11KODk7r5hGv8gM2v7co+RKi9d9soVoBxEboaOJoAnEB0zC8u7/qmDIPEs7nwHBDdVro47gZwowfuN0rZn1hSOXtjXbKVZ4ieA8u4CmtBmWnyPv8Qlo3e+pWKAYekTY0aIilu6PeLAbrXcUTWdVMZX7IlFHp47lXLV3AQeS2NZsG3y9ER7DtSlrGRlB0chz3CD0EYpWOIsV4D2hIHLXXzAFIWfaU2lNvcNIg7OkYyF+Xj0jKjXlgIoAvP+xim654v1lixz+/xCOCo/xEKzGghhAs0e8NWmWu4kJbK60GE3HMYBv/ACWKFDzcuFabefiJuAdzMWgYIjG0PiPU0Wj3lzznAS0ZW6+CC8HlmaIWusa7hzLRn1iYKt28y8s85fYgU4oeeJVgAH37jsCU/MGt3DuLYQNHLMxCsBxDHna6+YAbpHEF0KswFcK2+0ALZYeoBDG14g1ga33FaIdl3BWoAYiw5PA/cDCSiw5gNu3RzLM2roPaCRph9o+KGCbCrIVqBVY8D9sfBFu4bFqm/FT+qr5miWx9a7lZwRb69waFX+viOxRUz4+kq7qka1U8HEoOQNH7h1mfDMTbL8dQSzWzxGRC0p6iIF2uWDTS9ryy1bBv9S7oQef1BBFREYMX1zGKseO/iAC1xRFV09QC3C4qeItY9DmVEVWEKtrllRxvy+0LEpTmI6qqlp5re4RNrfnqLKo0fERGa4Ov3PGG8HuxgoMDYK4QvJeePSVXb83qP0bXLwcwAOjbEWh55jLSDcGyBPOJeLr+3KgXIJklfBzLErbgOOolQFcpx8wjgK/iNoFPmU0HLmEnNt74iA5olsGAryl19D17jIGl1KE6MQEqvZAbuscHHcEFitH94iUBtfy/qDa1cz1KrVdpePmK5Payi68/MZNHlmOTY/2wBRQFktLo8e8qF9Gj9wsqW13EXlTUpXs/EvAjfDxHNyr5ZUgivjmOZWzcSxC9xbsBFahpjW2FYfW4eRj55f1EvsHffRMmUDH8S6aNHHzLqTTkH3gWDDxz3LMvL/Yim2rQxcc+owit8/uUAC9qNdzRyjFLoOX2jBYAH4iEMHXfrLofU579ITb4wHMUWentMOQVwrP1+sBuFv6SiFW3H81rxGOmoYqqsOfMRoit45io9EMxivEVKVeipkrz5D/YOorRFv2wCBYKYBrZvqMALd+teswRo3MQg+YlRf3hmgAYjAbBov8AMRQY8HuywtWGu5dMlaqtdQgUKMdQ7GS/ER8rd+san1XAClAY1f0lhvVxoKBtChaj9/WXBTw7hNhVeDzNAJquIyYPUqBl+PdiDQp1LdRe2tekS7Z0u669YbF2NrxBINuWWr+x/wAilrpn14gHShqI6AaCXHIGQ19YdmLaOWXxt8JdsWt8HpLmhc2Y8njth8ovLiaO1cwqWizLzszFOvq8QSuGc+8bUpz38S5y8uJsVBHS8A4OCPgfOD9xpYVp++ox2rivaVSp8+o2QXj0+ZQgPkvXcxVc5Zcg1q/1DCic/aUxR5ESgKB1q4SqwZDn4gx78ECWWcF6ggQQFg3Sz9wsTy91KNGVhZWA4M5zM1zKy8RVwAZZqhpn1mdw/wBRRUKeI45wFB/eYABW8TMvQc9wkiqx9ZZyivPsQUxhl/m4FBxefSDSr75ZSJvt5+IxLYWn14lEGqqEBbWaPdl1HFYOfWWldmCWtbdB7R/c8fzmZpmwPSFqWrlzCNNu/wBSuzLwt/5EwvrDAFBlgik5f1yoVlk7mQgrxGZeqoghRbs4QBGnwzpjN2qfvAFm3n2hk3jKeIpFhzfXcCrADd89/wCy4qh28v6j3JBj1eJscAiRD4DqUC2vB7xHC3xEVWsQLNzXpLyK/wDEpW1bu9dx5ucrUPAp5gi1n59pcWXkX2lbtjKwpRRzdX6ygE8+4zG31b6mSWg4+IibaDRxNIx4P3Li5WuUtIUYolARmfZ4jYBrvj6cxsNqbRllZ5zm3mUOg2xqho6OIhAUHXPrANTTR5/5MhLujxfpEBNt0ddS0MdyGbo/4hXoV/KVve11+DqEKF5y/Uzk421CsUAW/uOclX7+selx/tAKrQf2JSKtVjqZFmv5gRKsYJY5OD6wbsnuXFx8n+blTLcv12wsUI2rm5Ru2y8zIBjnnqXJVRNlgc/7GTw+IKc+DWYqha4OuoABbHGvmCv+H1hUDL5c9woYvyvmUWinC8+kZGhtr+zFaqDaXqsHTzGebyp5myujBX+S0+GA4lW0tLDghAGyaq1OYYbLfpxNwzp669YHiM7eJScMZX/Y1l3y8/ENtA9b6iBeBKr9R9gOiEhFbB+4Au7u6OYqP2fqLnSx4NTLALPsQsFl3+5Rg54f2ISKxdLz0TLAgczS8desCBmv8gIpy3i+4ycA/LGrd0z6sYlQBX/CYkcNH95mQDZYfuBZcjHJlkstiADM
```

Comment:
Do not include the dist folder in your PR (I recommend making another Commit with the old dist folder).
Vue is automatically built via our CI.